90captive_portal_test

```diff
@@ -1,49 +1,44 @@
 #!/bin/sh -e
-# Script to dispatch NetworkManager events
 #
 # Runs io.elementary.capnet-assist helper on walled garden networks.
 # See NetworkManager(8) for further documentation of the dispatcher events.
 
-PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-
-if [ -x "/usr/bin/logger" ]; then
-    logger="/usr/bin/logger -s -t CapNetAssist"
-else
-    logger=":"
-fi
+msg() {
+    if [ -n "$(command -v logger)" ]; then
+        logger -s -t io.elementary.capnet-assist $@ || :
```

Comment:
I think you have to double quote `$@` to `"$@"`

Reply:
logger accepts multiple arguments, but it is probably a good idea to quote it either way

Reply:
I see now why I chose the former... I am passing arguments to the function that should be individual arguments to logger. Quoting will make these several arguments a single argument ("-p user.debug message" rather than "-p" "user.debug" "message") and break logger's arg parsing.